### ISSUE TITLE
Allow array for dynamic routing

### ISSUE THREAD

**Issue by pvullioud** [slimphp/Slim#951]
We can configure multiple HTTP methods like this

``` php
$app->map('/foo/bar', function() {
    echo "I respond to multiple HTTP methods!";
})->via('GET', 'POST');
```

I propose to also allow an array 

``` php
$methods = array('GET','POST');
$app->map('/foo/bar', function() {
    echo "I respond to multiple HTTP methods!";
})->via($methods);
```

Dynamic routing will be easier : 
http://help.slimframework.com/discussions/questions/812-configuring-routes-dynamically

**Comment by silentworks** [comment]
Already fixed on develop branch https://github.com/codeguy/Slim/commit/86654ba38ffa5f9c45422d8e8268a9698080c182 we will align the branches properly soon and probably move some of these changes to v2.

**Comment by pvullioud** [comment]
Great, thanks

**Comment by silentworks** [comment]
This have now been merged into master branch 77ab910.
